docs/about/overview/range-test.mdx

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
I think this is great! This type of content may eventually be moved to the showcase section. But I'm glad you pulled the data here.